src/plugins/gs-plugin-external-apps.c

```diff
@@ -626,8 +626,8 @@ gs_plugin_get_app_external_runtime (GsPlugin *plugin,
 
 	priv = gs_plugin_get_data (plugin);
 
-	full_id = g_strdup_printf ("%s:%s",
-				   gs_flatpak_get_prefix (priv->flatpak),
+	full_id = g_strdup_printf ("%s%s",
+				   gs_flatpak_get_prefix (priv->usr_flatpak),
```

Comment:
The change from `priv->flatpak` to `priv->usr_flatpak` looks like something for the next commit.